app/assets/javascripts/components/gtm-form-listener.js

```diff
@@ -0,0 +1,35 @@
+function GTMFormListener ($form, $dataLayer) {
```

Comment:
The dollar convention is used for HTML elements so shouldn't be on dataLayer as far as I'm aware.

(it's not my favourite convention)

Reply:
ah, this PR is mostly cargo culting 😟

Reply:
Fixed in 2b3f06b78e86bdb01b2282793ac191ab6803ed9a